vm/vm.go

```diff
@@ -204,7 +204,12 @@ func (inst *Instance) MonitorExecution(outc <-chan []byte, errc <-chan error,
 			if time.Since(lastExecuteTime) < noOutputTimeout {
 				break
 			}
-			if inst.Diagnose() {
+			diag, wait := inst.Diagnose()
+			if len(diag) > 0 {
+				mon.output = append(mon.output, []byte("DIAGNOSIS:\n")...)
```

Comment:
Strings are directly appendable to byte slices, so we can do just:
```
mon.output = append(mon.output, "DIAGNOSIS:\n"...)
```

Reply:
huh, TIL. Done.